src/test/integration/IntegrationDeployer.t.sol

```diff
@@ -868,9 +767,11 @@ abstract contract IntegrationDeployer is ExistingDeploymentParser {
     }
 
     /// @dev By default will have a assetType of HOLDS_LST
-    function _dealRandAssets_M1(User user) internal returns (IStrategy[] memory, uint[] memory) {
+    function _dealRandAssets_M1(
```

Comment:
Can we change `_dealRandAssets_M1` as well as `_dealRandAssets` to deploy via the StrategyFactory contract. We'll need to add this to the `_setUpLocal` to deploy the StrategyFactory as well and set it as the whitelister when initializing the StrategyManager.

We should mimic what is actually on Mainnet and never got around to adding this to integration tests.